### PR TITLE
Clarify docstring for twisted.internet.udp.Port.write()

### PR DESCRIPTION
**Pull request by rodrigc**
https://twistedmatrix.com/trac/ticket/8635

### CODE REVIEW

File: twisted/internet/udp.py [hunk 1]
```diff
@@ -323,7 +323,7 @@ def writeSequence(self, seq, addr):
         @param addr: A tuple of (I{stringified IPv4 or IPv6 address},
             I{integer port number}); can be L{None} in connected mode.
         """
-        self.write("".join(seq), addr)
+        self.write(b"".join(seq), addr)
```
Comment (scop) [inline twisted/internet/udp.py:327]: This looks like a functional change, and most likely a good one that would fix this method on Python 3. But it doesn't have test coverage, could you add a test case for it? Do you think it would be good to split it into a separate PR?

Comment (rodrigc) [inline twisted/internet/udp.py:327]: @scop you are right.  I am going to take this part out and submit a separate ticket for that, and add a unit test for writeSequence().

File: twisted/internet/udp.py [hunk 2]
```diff
@@ -311,6 +311,18 @@ def write(self, datagram, addr=None):
...
+
+        @param seq: The data that will make up the complete datagram to be
+            written.
+        @type seq: L{list} of L{bytes}
+
+        @type addr: L{tuple} containing L{str} as first element and L{int} as
+            second element, or L{None}
+        @param addr: A tuple of (I{stringified IPv4 or IPv6 address},
+            I{integer port number}); can be L{None} in connected mode.
+        """
```
Comment (scop) [inline twisted/internet/udp.py:325]: The `addr` docs seem like they're copied from `write` -- I'm not that familiar with the tools and doc format, but is there a convention or a way that would make it possible just point at other docs (here something like "see the addr parameter of write") instead of having to copy/paste? If not, and if this matches the current convention, no objections here.

Comment (rodrigc) [inline twisted/internet/udp.py:325]: @scop I changed it to an "iterable of {bytes}"

Comment (rodrigc) [inline twisted/internet/udp.py:325]: Yes, I copied the docs.  I don't know how to link it like you said.
As long as it is correct, I suppose it is OK?

Comment (scop) [inline twisted/internet/udp.py:325]: Sure

Comment (rodrigc) [inline twisted/internet/udp.py:325]: I've incorporated your feedback, just to confirm, are you OK if I merge this?
Thanks.

Comment (scop) [inline twisted/internet/udp.py:325]: Yes, ticket updated per review process instructions (not sure if I should ack PR's in Trac or here or both)
